prereise/gather/hiflddata/data_trans.py

```diff
@@ -0,0 +1,645 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+
+import csv
+import json
+import os.path
+import zipfile
+from collections import defaultdict
+
+import networkx as nx
+import pandas as pd
+from haversine import Unit, haversine
+
+Max_Value = 3000
+Min_Value = 0
```

Comment:
These variables are being reassigned from within `InitKV`, if they do not need to be global variables I suggest that they be removed.

Reply:
Yes, I will remove them